packages/prosemirror-lwdita-demo/src/example.ts

```diff
@@ -6,19 +6,25 @@ import jsonDocLoader from "./doc";
 import { menu, shortcuts } from "@evolvedbinary/prosemirror-lwdita";
 import { githubMenuItem, openFileMenuItem, publishFileMenuItem, saveFileMenuItem} from "./demo-plugin";
 import { history } from "prosemirror-history";
-import { doubleClickImagePlugin, processRequest } from '@evolvedbinary/prosemirror-lwdita'
+import { doubleClickImagePlugin, processRequest, fetchAndTransform } from '@evolvedbinary/prosemirror-lwdita'
 
 const schemaObject = schema();
 
 /**
  * Process the URL parameters and handle the notifications
  */
-const urlParams = processRequest();
-// TODO(YB): fetch and render the file from the URL
+const urlParams = processRequest() as { key: string, value: string }[];
 
 // if the user passes a file in the URL, load that file
 // otherwise load the default file
-const loadJsonDoc = jsonDocLoader;
+let loadJsonDoc = jsonDocLoader;
+if(urlParams) {
+  const ghrepo = urlParams.find((param) => param.key === 'ghrepo');
+  const source = urlParams.find((param) => param.key === 'source');
```

Comment:
This logic should be moved into `request.ts` in `prosemirror-lwdita`. 
We shouldn't process the parameters here but instead, retrieve the values from the request module.
Plus, `ghrepo` and `source` are defined in a constant there and we expect to maintain them only there in one single place. Every update of the parameter names would require to also update them in this "demo" package.

Reply:
The way `urlParqams` variable is structured require us to use the array find method, and we need to provide a static key.
As of now I don't see another way of getting `ghrepo`, `source`.

Reply:
Fixed with https://github.com/evolvedbinary/prosemirror-lwdita/pull/394/commits/f7f7ee911aeeb7e35718c54846a8daafefa19477